src/components/App/App.component.jsx

```diff
@@ -30,13 +31,18 @@ function App() {
     };
   }, []);
 
+  const handleSearch = (event) => {
+    setSearch(event.target.value);
+    console.log(search);
```

Comment:
Please try to remove the console.logs for your PRs

Reply:
Removed